src/strategies/github-strategy.mjs

```
@@ -0,0 +1,72 @@
+import "../mongoose/schemas/github-user.mjs";
+import { GithubUser } from "../mongoose/schemas/github-user.mjs";
+import { User } from "../mongoose/schemas/user.mjs";
+
+passport.serializeUser((user, done) => {
+    console.log(`Inside serialize user`);
+    console.log(user);
+    done(null, user.id);
+});
+
+passport.deserializeUser(async (id, done) => {
+    console.log(`Inside deserializer`);
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)